### PR TITLE
[hueemulation] Consider property discoveryIps if configured (#6662)

### LINKED ISSUE #6662: [hueemulation] hueemulation (openhab 2.5) not working in docker
Hi,

i am running openhab based on the official docker image on a raspi.
I use the hueemulation-service to integrate my openhab-items into alexa.
Until my update to openhab 2.5 everything was working fine, but after the update i cannot connect them together anymore.

I removed all devices that are configured by openhab in alexa and now i am trying to link them again, but the hueemulation-bridge is not found by my devices.
This is my config:

```
org.openhab.hueemulation:discoveryIps=192.168.4.200
org.openhab.hueemulation:temporarilyEmulateV1bridge=false
org.openhab.hueemulation:createNewUserOnEveryEndpoint=true
org.openhab.hueemulation:pairingEnabled=true
```

i also tried to set temporarilyEmulateV1bridge to true but that does not make any difference.

openhab is also avaiable on port 80, as adviced by the hueemulation documentation.
What wonders me is that /description.xml show the following URLBase:

```
<root>
<specVersion>
<major>1</major>
<minor>0</minor>
</specVersion>
<URLBase>http://127.0.0.1:80/</URLBase>
```

Is that a problem? how can i change that?

And do i need to select Philips Hue V1 or V2 bridge in the alexa app, when i try to connect them?

### PR DESCRIPTION
**Pull request by marbon87**
Fixes #6662

### CODE REVIEW

**Comment by marbon87** [comment]
hi @davidgraeff , would you take a look?

File: bundles/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/ConfigStore.java [hunk 1]
```diff
@@ -246,6 +252,17 @@ public void modified(Map<String, Object> properties) {
         }
     }
 
+    private String getConfiguredHostAddress(InetAddress configuredAddress){
```
Comment (kaikreuzer) [inline bundles/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/ConfigStore.java]:
```suggestion
    private String getConfiguredHostAddress(InetAddress configuredAddress) {
```

**Comment by kaikreuzer** [comment]
@marbon87 In order for us to merge, you would need to sign-off your contribution, see https://www.openhab.org/docs/developer/contributing.html#sign-your-work.

It would be enough if you post the sign-off line as a comment here on the PR.

**Comment by marbon87** [comment]
Signed-off-by: Mark Bonnekessel <bonnekessel@mailbox.org>

**Comment by kaikreuzer** [comment]
That was quick, many thanks!

**Pull request merged**